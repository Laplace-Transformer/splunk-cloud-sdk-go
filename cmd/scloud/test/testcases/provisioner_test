provisioner create-invite --email aa@splunk.com
provisioner get-invite --invite-id i5e603d1366957b00069575fe
provisioner list-invites
provisioner delete-invite --invite-id i5e603d1366957b00069575fe
provisioner list-tenants
provisioner update-invite --invite-id i5e603d9225fe920006b12c8e --action accept
