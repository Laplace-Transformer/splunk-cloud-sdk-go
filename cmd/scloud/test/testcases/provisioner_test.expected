#testcase: provisioner create-invite --email aa@splunk.com
REQUEST URL:provisioner/v1beta1/invites
REQUEST BODY:{{"email":"aa@splunk.com"}}

#testcase: provisioner get-invite --invite-id i5e603d1366957b00069575fe
REQUEST URL:provisioner/v1beta1/invites/i5e603d1366957b00069575fe
REQUEST BODY:<nil>

#testcase: provisioner list-invites
REQUEST URL:provisioner/v1beta1/invites
REQUEST BODY:<nil>

#testcase: provisioner delete-invite --invite-id i5e603d1366957b00069575fe
REQUEST URL:provisioner/v1beta1/invites/i5e603d1366957b00069575fe
REQUEST BODY:<nil>

#testcase: provisioner list-provision-jobs
REQUEST URL:provisioner/v1beta1/jobs/tenants/provision
REQUEST BODY:<nil>

#testcase: provisioner list-tenants
REQUEST URL:provisioner/v1beta1/tenants
REQUEST BODY:<nil>

#testcase: provisioner update-invite --invite-id i5e603d9225fe920006b12c8e --action accept
REQUEST URL:provisioner/v1beta1/invites/i5e603d9225fe920006b12c8e
REQUEST BODY:{{"action":"accept"}}

#testcase: provisioner create-provision-job --apps "newapp,newapp2"
REQUEST URL:provisioner/v1beta1/jobs/tenants/provision
REQUEST BODY:{{"apps":["newapp","newapp2"]}}

#testcase: provisioner get-provision-job --job-id 5e603e3425fe920006b12c8f
REQUEST URL:provisioner/v1beta1/jobs/tenants/provision/5e603e3425fe920006b12c8f
REQUEST BODY:<nil>

